src/views/VehicleConfig.vue

```diff
@@ -139,16 +139,35 @@
               @update:model-value="updateState('openWB/vehicle/' + vehicleId + '/charge_template', $event)"
             />
             <hr />
-            <div v-if="$store.state.mqtt['openWB/optional/rfid/active'] === true && !installAssistantActive">
-              <openwb-base-array-input
-                title="Zugeordnete ID-Tags"
-                :model-value="$store.state.mqtt['openWB/vehicle/' + vehicleId + '/tag_id']"
-                @update:model-value="updateState('openWB/vehicle/' + vehicleId + '/tag_id', $event)"
-              />
-              <openwb-base-alert subtype="info">
-                Die hier eingetragenen ID-Tags dienen ausschließlich der Fahrzeugzuordnung.<br />
-                <vehicle-id-wiki-hint />
-              </openwb-base-alert>
+            <div v-if="!installAssistantActive">
+
+            <openwb-base-heading> Fahrzeugzuordnung per ID-Tags </openwb-base-heading>
+              <div v-if="$store.state.mqtt['openWB/vehicle/' + vehicleId + '/tag_id'].length > 0">
+                <openwb-base-alert subtype="info">
+                  Einstellungen zur Fahrzeugzuordnung finden sich unter
+                  <router-link to="/IdentificationConfig"> Einstellungen - Identifikation </router-link>.
+                  <div v-if="$store.state.mqtt['openWB/optional/rfid/active'] === false">
+                    Aktuell ist die Option in den Einstellungen deaktiviert.
+                  </div>
+                  <div v-else>
+                    Die Option ist aktiv. Das Fahrzeug lässt sich per ID-Tag automatisch
+                    einem Ladepunkt zuordnen.
+                  </div>
+                  Dem Fahrzeug sind folgende ID-Tags zugeordnet:
```

Comment:
Warum kann ich die Tags in der Fahrzeug-Karte nicht bearbeiten, Standard nach Abstecken im Ladeprofil aber schon?